package.json

```diff
@@ -42,13 +42,14 @@
     "pretest": "npm run compile"
   },
   "dependencies": {
-    "@google-cloud/common": "^0.19.2",
+    "@google-cloud/common": "^0.20.0",
     "@grpc/proto-loader": "^0.1.0",
     "dot-prop": "^4.2.0",
     "duplexify": "^3.6.0",
     "extend": "^3.0.1",
     "grpc": "^1.12.3",
     "is": "^3.2.1",
+    "request": "^2.87.0",
```

Comment:
We shouldn't need this. We do our requests through the internal "request" method. Were we getting an error?

Reply:
I wasn't, but I also don't entirely trust the system tests in this project (there are no system-tests).  Thanks for the heads up :)